src/core/components/Conversation/Conversation.tsx

```diff
@@ -26,15 +26,11 @@ const ConversationComponent = ({ onRendered }: ConversationProps) => {
   );
 
   const { messages } = useMessageHistory();
-  // todo: re-implement this to make it work with the ImageCarousel
-  // const [loadedImages, setLoadedImages] = useState<Record<string, boolean>>({});
+  const [isLoaded, setIsLoaded] = useState(false);
```

Comment:
We don't need to track loading states for individual files, just one state for when the DOM can be updated.